### PR TITLE
feat: update navbar UI per Figma

### PR DESCRIPTION
**Pull request by ZachJW34**
- Closes [UNIFY-948](https://cypress-io.atlassian.net/browse/UNIFY-948)

### User facing changelog
Update navbar UI to match Figma mocks

### Additional details
The testing type switcher has moved into the header and some colors have been tweaked

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/151261294-94c55eaa-c24b-4714-bcba-2f25bf452ef0.mov

### PR Tasks

- [x] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/frontend-shared/src/gql-components/HeaderBarContent.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="bg-white border-b border-b-gray-100 py-15px px-6">
```
Comment (elevatebart) [inline packages/frontend-shared/src/gql-components/HeaderBarContent.vue:2]: This height should be dictated by the grid container right?

**Review by elevatebart — DISMISSED** [review]
I like this UI a lot

**Review by elevatebart — APPROVED** [review]
I want this in fast !!!

**Comment by ZachJW34** [comment]
@elevatebart reverted the `h-full`, it was causing issues in tests and I think it's fine that it is fixed at 64px, makes it consistent across wherever it's used (launchpad and app).

**Review by elevatebart — APPROVED** [review]

**Review by elevatebart — APPROVED** [review]

**Pull request merged**